src/client/render/core.h

```diff
@@ -21,6 +21,11 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #pragma once
 #include "irrlichttypes_extrabloated.h"
 
+namespace irr
+{
+	class IrrlichtDevice;
+};
```

Comment:
```suggestion
}
```

Reply:
Apparently I've been writing too many structs and classes recently.